packages/components/ContextualMenu/package.json

```diff
@@ -30,7 +30,8 @@
     "@fluentui-react-native/text": ">=0.9.12 <1.0.0",
     "@fluentui-react-native/tokens": ">=0.9.8 <1.0.0",
     "@uifabricshared/foundation-composable": ">=0.8.4 <1.0.0",
-    "@uifabricshared/foundation-settings": ">=0.9.2 <1.0.0"
+    "@uifabricshared/foundation-settings": ">=0.9.2 <1.0.0",
+    "react-native-svg": "^12.1.1"
```

Comment:
I think we prefer `react-native-svg` as a peer dependency, so that we don't accidentally bring in our own version of this package in case a client already packages it. It's similar to how we moved react/react-native/etc to peerDependencies